docs/docs/sources/chatplugin/customization.md

```diff
@@ -2,3 +2,62 @@
 title: Basic & advanced customization
 sidebar_label: Basic & advanced customization
 ---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## How to customize the Airy Chat Plugin
+
+## Basic customization
```

Comment:
This section looks nice but I think we should also provide a basic table with "option name" "option description" and a little snippet of code example for it